proto/mls/message_contents/association.proto

```diff
@@ -21,6 +23,14 @@ message Eip191Association {
   string iso8601_time = 4;
 }
 
+// LegacyCreateIdentityAssociation is used when a v3 installation key
+// is signed by a v2 identity key, which in turn is signed via a
+// 'CreateIdentity' wallet signature
+message LegacyCreateIdentityAssociation {
+  RecoverableEcdsaSignature signature = 1;
+  SignedPublicKey legacy_create_identity_key = 2;
```

Comment:
The creation time and account address can be derived from the nested `SignedPublicKey` proto